### PR TITLE
chore: event sync to lark

### PR DESCRIPTION
**Pull request by huntye1**
## Release Notes

- **New Features**
  - Introduced a new GitHub Action for synchronizing GitHub events (pull requests, issues, CI updates) with Lark groups.
  - Added an event synchronization workflow that triggers on specific GitHub events.

- **Documentation**
  - Created a README for the event sync action, detailing its functionalities and usage.

- **Tests**
  - Implemented unit tests for event handlers and the handler factory to ensure correct event processing and notification sending.

- **Chores**
  - Added TypeScript configuration files and various supporting files to enhance project structure and build processes.
  - Removed the previous notification workflow for Feishu.

### CODE REVIEW

File: .github/actions/event-sync/lib/package.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Tecvan-fe) [inline .github/actions/event-sync/lib/package.json:1]: 产物也要提交进去？

File: rush.json [hunk 2]
```diff
@@ -390,6 +390,10 @@
       "packageName": "@coze-infra/fs-enhance",
       "projectFolder": "infra/utils/fs-enhance"
     },
+    {
+      "packageName": "@coze-infra/event-sync-action",
+      "projectFolder": ".github/actions/event-sync"
```
Comment (Tecvan-fe) [inline rush.json:395]: .github 是个特殊目录，别放源码；源码放在别的地方，.github/actions 放 action.yml 引用过去，应该就OK了吧？

File: .github/actions/event-sync/lib/package.json (hunk 1 above)
Comment (huntye1) [reply .github/actions/event-sync/lib/package.json:1]: 实测产物运行起来比较快，不然还要在actin里面安装依赖。官方也推荐打包产物。

File: rush.json (hunk 2 above)
Comment (huntye1) [reply rush.json:395]: OK

**Review by Tecvan-fe — APPROVED** [review]

**Pull request merged**